docker-compose.yaml

```diff
@@ -51,6 +51,20 @@ services:
         ipv4_address: 172.25.0.10
     volumes:
       - ./certs:/etc/letsencrypt:rw
+      
+   
+  postgres:
```

Comment:
how can i run docker-compose locally?

Reply:
because if i just run `docker-compose up -d` it will build me an image with unencrypted data